src/components/ProductView/product-detail.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const ProductDetail = () => {
+  return (
+    <section className="product-detail">
```

Comment:
Let's use `div` for this first.

Reply:
Do you mean have `section` inside another `div` or use `div` instead of `section`